src/components/Location.astro

```diff
@@ -140,56 +140,56 @@
                 background-image: url("/images/vakwerkhuis.jpg");
             }
 
-            @include wider_than(sm) {
+            @include breakpoint.wider_than(sm) {
                 grid-area: b;
             }
         }
 
         .image:nth-of-type(3) {
             display: none;
 
-            @include wider_than(sm) {
+            @include breakpoint.wider_than(sm) {
                 aspect-ratio: 16 / 9;
-                background-color: $color-yellow-400;
+                background-color: variables.$color-yellow-400;
                 grid-area: c;
                 display: flex;
                 align-items: center;
                 justify-content: center;
             }
 
             &::after {
-                @include wider_than(sm) {
-                    background-color: $color-yellow-500;
+                @include breakpoint.wider_than(sm) {
+                    background-color: variables.$color-yellow-500;
                     content: "";
                     inset: 0 50% 0 0;
                     position: absolute;
                 }
             }
 
             &:hover .ferris {
-                @include wider_than(sm) {
-                    translate: 0;
-                }
-
                 /* This timing is for the entering animation */
                 transition: translate 300ms 200ms ease-out;
-            }
 
-            &:hover .ferris-arms {
-                @include wider_than(sm) {
```

Comment:
moved around some things because scss deprecated mixing blocks and properties (properties now need to come first)